Filmap/Filmap/MoviePage.xaml.cs

```diff
@@ -39,12 +47,83 @@ private void button_Click(object sender, RoutedEventArgs e)
 
         private void button1_Click(object sender, RoutedEventArgs e)
         {
-
+            SaveMovie();
         }
 
         private void Pivot_Loaded(object sender, RoutedEventArgs e)
         {
-
+            
         }
+
+        private async void SaveMovie()
+        {
+            // instancia um cliente http para acessar a api
+            HttpClient httpClient = new HttpClient();
+            httpClient.BaseAddress = new Uri((App.Current as App).filmapApiUrl);
+
+            // instancia geolocalizador para pegar posicao do usuario
+            Geolocator geo = new Geolocator();
+            geo.DesiredAccuracy = PositionAccuracy.High;
+            var position = await geo.GetGeopositionAsync();
+
+            // get latitude and longitude
+            String lat = Convert.ToString(position.Coordinate.Point.Position.Latitude);
+            String lgn = Convert.ToString(position.Coordinate.Point.Position.Longitude);
+
+            //MessageBox.Show(Regex.Match(searchResultMovieLocal.imdbID, @"\d+").Value);
+
+            String omdbId = Regex.Match(searchResultMovieLocal.imdbID, @"\d+").Value;
+
+            
+
+            // prepara os parametros para mandar no post request
+            var credentials = new FormUrlEncodedContent(new[] {
+                new KeyValuePair<string, string>("omdb", searchResultMovieLocal.imdbID), 
+                new KeyValuePair<string, string>("watched", "1"), // zero or one
+                new KeyValuePair<string, string>("lat", lat),
+                new KeyValuePair<string, string>("lng", lgn)
+            });
+
+            String token = (App.Current as App).accessToken;
+
+            // faz um post request e espera o resultado asincronamente
+            var response = await httpClient.PostAsync("/films?token=" + token, credentials);
```

Comment:
Isso e uma das coisas que fica pra quando a gente acabar tudo e comecar a fazer melhorias. Por enquanto vai ficar assim for the sake of simplicity.